src/copilot.ts

```diff
@@ -28,19 +28,20 @@ const completionSpec: Fig.Spec = {
           description: "Deploy service/job to a 'test' environment",
         },
         {
-          // TODO: Is there a way to make this mutually exclusive with `-i`?
           name: ["-d", "--dockerfile"],
           description: "Path to the Dockerfile",
           args: {
             name: "string",
           },
+          exclusiveOn: ["-i", "--image"],
```

Comment:
I'm pretty sure `-i` is enough
@QuiiBz am I right?
```suggestion
          exclusiveOn: ["-i", "--image"],
```